README.md

```diff
@@ -112,6 +112,7 @@
     b) [LLVM](https://llvm.org/) (LLVM13)
     - Install [LLVM-13.0.0-win64.exe](https://github.com/llvm/llvm-project/releases/tag/llvmorg-13.0.0). Change the LLVM install path to `C:\LLVM`, and add LLVM path `C:\LLVM\bin` in PATH environment for CLANG build on Windows.
     - LLVM13 works good for clang and [libfuzzer](https://llvm.org/docs/LibFuzzer.html) build. Other versions are not validated for clang build.
+    - The Visual Studio is needed for clang build. If choose to install LLVM, please also install the Visual Studio.

```

Comment:
would you please describe the reason?

I think `nmake` is needed. Anything else?